packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/KotlinInteropModuleRegistry.kt

```diff
@@ -26,9 +28,16 @@ class KotlinInteropModuleRegistry(
   fun hasModule(name: String): Boolean = registry.hasModule(name)
 
   fun callMethod(moduleName: String, method: String, arguments: ReadableArray, promise: Promise) {
-    registry
-      .getModuleHolder(moduleName)
-      ?.call(method, arguments, promise)
+    try {
+      requireNotNull(
+        registry.getModuleHolder(moduleName)
+      ) { "Trying to call `$method` on the nom-existing module `$moduleName`" }
```

Comment:
Do we want to wrap names in the single/double quotes or backticks ``'moduleName' "moduleName" `moduleName` ``? cc @tsapeta 
```suggestion
      ) { "Trying to call '`$method`' on the non-existing module '`$moduleName`'" }
```